src/components/common/Actions.vue

```diff
@@ -166,8 +166,16 @@ export default class Actions extends Mixins(DateMixin, FilingTemplateMixin, Name
     // from processNameRequest() in App.vue. This method shows a generic message if the Name
     // Request is invalid, and clicking OK in the pop up redirects to My Business Registry.
     if (this.getNameRequestNumber) {
+      console.log('hello')
       try {
-        await this.validateNameRequest(this.getNameRequestNumber)
+        if (this.getNameRequest.applicant) {
+          await this.validateNameRequest(
+            this.getNameRequestNumber,
+            this.getNameRequest.applicant.phoneNumber,
+            this.getNameRequest.applicant.emailAddress)
+        } else {
+          await this.validateNameRequest(this.getNameRequestNumber)
+        }
```

Comment:
Haha, great solution! Did you test all the flows that allow a NR?

Reply:
@PaulGarewal 😉

Reply:
@JazzarKarim the hero for this one

Reply:
So, did you test all the flows?

Reply:
I am currently for when both email and phone number are entered, just need to create a new nr

Reply:
Remember that user only needs to enter phone or email (not both).

I was going to mention it above, since you always call validateNameRequest() with both, but I think if the user left one blank (empty string or null?) then validateNameRequest() and the API endpoint will work anyway.

I meant the different flows such as Change of Name Filing, Conversion Filing, Alteration Filing, Correction Filing.

Reply:
Yes if one is empty it will work.

Okay I am on it with the different flows.

Reply:
All flows that can be tested have been tested 👍